cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/scheduler/AwsBackoffSyncPollingScheduler.java

```diff
@@ -20,14 +22,16 @@
 
     private static final int POLLING_INTERVAL = 5;
 
-    private static final int MAX_POLLING_INTERVAL = 10;
+    private static final int MAX_POLLING_INTERVAL = 50;
 
     private static final int MAX_POLLING_ATTEMPT = 1000;
 
-    private static final int FAILURE_TOLERANT_ATTEMPT = 3;
+    private static final int FAILURE_TOLERANT_ATTEMPT = 10;
 
     private static final String THROTTLING_ERROR_CODE = "Throttling";
 
+    private static final Random RANDOM = new SecureRandom();
```

Comment:
Why secure random? Why not threadlocalrandom?

Reply:
I just looked at what is the general practice in this codebase. All the places I saw it was secure random. No strong takes here. Happy to change it.

Reply:
Fixed it! Thanks!